.gitignore

```diff
@@ -1,36 +1,37 @@
-# Ignore IntelliJ IDEA files and folders
-*.idea
-*.iml
-*.ipr
-*.iws
-*.ids
-*.log
-
-conf
-
-/out
-/.gradle
-
-# Ignore Eclipse files and folders
-*.project
-*.settings
-*.classpath
-
-# Ignore NetBeans files and folders
-**/nbproject
-**/nb-configuration.xml
-
-# Ignore Maven files and folders
-**/target
-
-# Ignore DB folder
-**/db
-
-# Ignore logs
-**/logs
-
-**/Thumbs.db
-/build/
-/dist/
-
-/gource/
+# Ignore IntelliJ IDEA files and folders
+*.idea
+*.iml
+*.ipr
+*.iws
+*.ids
+*.log
+
+conf
+
+/out
+/.gradle
+
+# Ignore Eclipse files and folders
+*.project
+*.settings
+*.classpath
+
+# Ignore NetBeans files and folders
+**/nbproject
+**/nb-configuration.xml
+
+# Ignore Maven files and folders
+**/target
+
+# Ignore DB folder
+**/db
+
+# Ignore logs
+**/logs
+
+**/Thumbs.db
+/build/
+/dist/
+
+/gource/
+/target/
```

Comment:
Target is already being ignored on line 24. Also this showed up as an entire file change, which is weird. Maybe you changed the encoding? Just remove this file from your commit since it doesn't change anything.